pkg/client/client.go

```diff
@@ -70,6 +71,7 @@ func NewClient(options ...Options) (*Client, error) {
 		Blocks:    NewBlocks(opts),
 		Wallet:    NewWallet(opts),
 		Alias:     NewAlias(opts),
+		Peers:     NewPeers(opts),
```

Comment:
Why the generic config options pattern is not implemented here?

Reply:
Such pattern adds and removes functionality depends on passed params, in my case i just initialize public property. I think my version is much clean and obvious.